src/components/search/curated/default-cta.tsx

```diff
@@ -29,6 +29,7 @@ const DefaultCTA: React.FC<{location: string}> = ({location}) => {
           priority
           quality={100}
           layout="fill"
+          objectFit="cover"
```

Comment:
`objectFit="cover"` used to avoid image shrinking:

![Image 2022-04-21 at 12 11 49 PM](https://user-images.githubusercontent.com/1519448/164421877-1b4a23b5-1743-41fc-8fcc-6e7a2523af2b.jpg)